packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -189,12 +181,19 @@ const team = computed(() => props.workspaceInfo.team.items || [])
 const isWorkspaceAdmin = computed(
   () => props.workspaceInfo.role === Roles.Workspace.Admin
 )
+const isWorkspaceGuest = computed(
+  () => props.workspaceInfo.role === Roles.Workspace.Guest
+)
 const actionsItems = computed<LayoutMenuItem[][]>(() => [
   [
     ...(isMobile.value
       ? [
-          { title: 'Move projects', id: ActionTypes.MoveProjects },
-          { title: 'Invite', id: ActionTypes.Invite }
+          ...(isWorkspaceAdmin.value
```

Comment:
Previously the mobile buttons where always shown, now they are hidden based on the same permissions as we use on desktop.